pyof/utils.py

```diff
@@ -5,8 +5,12 @@
 """
 from pyof import v0x01, v0x04
 from pyof.foundation.exceptions import UnpackException
+from pyof.v0x01.asynchronous.error_msg import BadActionCode as BadActionCode01
 from pyof.v0x01.common import utils as u_v0x01  # pylint: disable=unused-import
+from pyof.v0x01.common.phy_port import PortConfig as PortConfig01
+from pyof.v0x04.asynchronous.error_msg import BadActionCode as BadActionCode04
 from pyof.v0x04.common import utils as u_v0x04  # pylint: disable=unused-import
+from pyof.v0x04.common.port import PortConfig as PortConfig04
```

Comment:
Suggestions:
1. Clean up imports, using only v0x01 and v0x04 already imported.
1. Add tests (possibly doctests).